test/compose/two_networks/tests.sh

```diff
@@ -1,7 +1,11 @@
 # -*- bash -*-
 
-podman container inspect two_networks_con1_1 --format '{{len .NetworkSettings.Networks}}'
+ctr_name="two_networks_con1_1"
+if [ "$TEST_FLAVOR" = "compose_v2" ]; then
+    ctr_name="two_networks-con1-1"
+fi
```

Comment:
Well that is is interesting, they changed the container name to using a dash but below for the network names they still use underscores.